docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: "3"
+services:
+  pyfahviewer:
+    build: .
+    ports:
+      - 5000:5000
```

Comment:
It's generally recommended to explicitly use a string here, [because](https://docs.docker.com/compose/compose-file/#ports):

>When mapping ports in the HOST:CONTAINER format, you may experience erroneous results when using a container port lower than 60, because YAML parses numbers in the format xx:yy as a base-60 value. For this reason, we recommend always explicitly specifying your port mappings as strings.